nbs/mint.json

```diff
@@ -119,6 +120,7 @@
             "models.nhits.html",
             "models.nlinear.html",
             "models.patchtst.html",
+            "models.rmok.html",
```

Comment:
Good catch! Is it also included in the other places where it should be included? See [our model contributing guide.](https://nixtlaverse.nixtla.io/neuralforecast/docs/tutorials/adding_models.html#4-add-the-model-to-the-documentation)

Reply:
Thanks, forgot about the sidebar!